ipa-core/src/protocol/hybrid/oprf.rs

```diff
@@ -293,4 +361,22 @@ mod test {
             }
         });
     }
+
+    #[test]
+    fn prf_indistinguishable_serialize_deserialize() {
+        run(|| async {
+            let world = TestWorld::default();
+            let mut rng = world.rng();
+            let report = PRFIndistinguishableHybridReport::<BA8, BA3> {
```

Comment:
given how often you use it, it is probably worth implementing `impl rand::distributions::Distribution<PRFIndistinguishableHybridReport> for rand::distributions::Standard {
}` for tests